Twilio-org/phonebank
# Ticket 111: Campaign preselected in my campaigns view

To test manually:
- Create a second campaign that the seeded user has not joined.
- Log in as the user (not admin) and observe that the first campaign is selected in the my campaigns view.
- Click the available campaigns tab and click the join button of the second campaign you created in the first step. Observe that campaign is now selected with the browser takes you back to the my campaigns view
